zeq/detail/eventDescriptor.h

```diff
@@ -18,19 +18,22 @@ namespace detail
 struct EventDescriptor : public boost::noncopyable
 {
     EventDescriptor( const std::string& restName, const uint128_t& eventType,
-                     const std::string& schema )
+                     const std::string& schema, const EventDirection eventDirection )
         : _restName( restName )
         , _eventType( eventType )
         , _schema( schema )
+        , _eventDirection( eventDirection )
     {}
     const std::string& getRestName() const { return _restName; }
```

Comment:
remove the functions and make the members public? They are const and it's a detail struct, so the getters are actually overkill :)